core/src/main/scala/com/karumi/shot/screenshots/ScreenshotsComparator.scala

```diff
@@ -67,6 +77,13 @@ class ScreenshotsComparator {
     }
   }
 
+  private def isRGBSimilar(p1: Pixel, p2: Pixel, colorTolerance: Int): Boolean = {
+    val absDiffRed   = Math.abs(p1.red - p2.red)
+    val absDiffGreen = Math.abs(p1.green - p2.green)
+    val absDiffBlue  = Math.abs(p1.blue - p2.blue)
+    absDiffRed <= colorTolerance && absDiffGreen <= colorTolerance && absDiffBlue <= colorTolerance
```

Comment:
What about alpha channel? Did you notice if there is any difference between the devices mentioned in the alpha channel as well when using images with some transparency?